Source/Tests/IRLoader.cpp

```diff
@@ -119,6 +120,25 @@ class IRCodeLoader final : public FEXCore::CodeLoader
   constexpr static uint64_t STACK_SIZE = 8 * 1024 * 1024;
 };
 
+class IRSyscallHandler: public FEXCore::HLE::SyscallHandler {
```

Comment:
`DummySyscallHandler` seems like a more apt name for this, as it stresses that this is just a NOP implementation (whereas the current name suggests some specific behavior that this module relies on).

Reply:
Done